eslint.config.mjs

```diff
@@ -10,7 +10,6 @@ export default [
       sourceType: 'module',
       globals: {
         ...globals.browser,
-        ...globals.es6
```

Comment:
I deleted this because `globals.es6` does not exist, it's `undefined`. es6 globals are already handled by this line:

https://github.com/openstreetmap/iD/blob/ba8d52e3c9ce7bf674693ef5144f7cf4a3c9bff1/config/eslint.config.mjs#L9